src/level-ext.adoc

```diff
@@ -123,15 +123,15 @@ NOTE: A capability with <<section_cap_level,CL>>=1 is _global_ and with <<sectio
 NOTE: if W=0 or C=0 then SL is irrelevant
 
 .{cheri_levels_ext_name} `LVLBITS=1` summary table for loaded capabilities
-[#cap_level_load_summary,width="100%",options=header,align=center,cols="1,1,1,1,1,6"]
+[#cap_level_load_summary,width="100%",options=header,align=center,cols="1,1,1,1,1,1,6"]
 |==============================================================================
-   4+|Auth cap field             | Data cap field |
-    h|*R*   h|*C* h|*EL* h|*CL* h| Tag           h|Action
-.5+.^|1 .5+.^| 1   | 1    | X    | X              |Load data capability unmodified
-              .4+.^| 0    | 1    | 1              |Load data capability with EL=0
-                          | 1    | 0              |Load data capability unmodified
-                     .2+.^| 0    | 1              |Load data capability with CL=0,EL=0
-                                 | 0              |Load data capability unmodified
+   4+|Auth cap field             2+| Data cap field |
+    h|*R*   h|*C* h|*EL* h|*CL*   h| Tag h| Sealed h|Action
+.5+.^|1 .5+.^| 1   | 1 .3+.^| X    | X    | X  .2+.^|Load data capability unmodified
+              .4+.^| 0             | 0    | N/A
+                              .3+.^| 1    | Yes     |Load data capability with `CL=min(auth.CL, data.CL)`, EL unchanged
```

Comment:
```suggestion
                              .3+.^| 1    | Yes     |Load data capability with `CL=min(auth.CL, data.CL)`, EL unchanged
```

I could use AND here instead, but `min` makes it portable to LVLBITS>1

Reply:
ok